### DISCUSSION TITLE
No warning for bypassing branch protection on github.dev on main branch

### DISCUSSION THREAD

**Discussion by Anton-4** [community/community#136665] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I opened the github.dev editor (with the . key) on the main branch without really thinking about it. After pushing my changes on github.dev I was startled to see that I pushed to the main branch and bypassed branch protection. On the github.com editor there is a clear warning and you have to manually select the "Bypass branch protection rules" in red font when pushing to the main branch, but on github.dev there was no such thing.

This is obviously dangerous so I recommend mitigating this with a similar dialog like on the github.com editor.

**Comment by crlgb** [comment]
Same here, this should be given more attention!
